OpenLIFUData/OpenLIFUData.py

```diff
@@ -466,13 +547,69 @@ def onAddNewSubjectClicked(self, checked:bool) -> None:
                 #Update loaded subjects view
                 self.updateSubjectSessionSelector()
     
+    @display_errors
+    def getSelectedSubjectSession(self, index: qt.QModelIndex) -> Tuple[str, str]:
+        """ Returns the subject or session (name, id) currently selected in the SubjectSessionView """
+        
+        name = self.subjectSessionItemModel.itemFromIndex(index.siblingAtColumn(0)).text()
+        id = self.subjectSessionItemModel.itemFromIndex(index.siblingAtColumn(1)).text()
+        
+        return (name, id)
+
     @display_errors
     def onAddVolumeToSubjectClicked(self, checked:bool) -> None:
         volumedlg = AddNewVolumeDialog()
         returncode, volume_filepath, volume_name, volume_id = volumedlg.customexec_()
         if not returncode:
             return False
-        self.logic.add_volume_to_database(self.currentSubjectID, volume_id, volume_name, volume_filepath)
+        
+        currentIndex = self.ui.subjectSessionView.currentIndex()
+        current_subject = self.getSelectedSubjectSession(currentIndex)
+        self.logic.add_volume_to_database(current_subject[1], volume_id, volume_name, volume_filepath)
```

Comment:
```suggestion
        _, current_subject_id = self.getSelectedSubjectSession(currentIndex)
        self.logic.add_volume_to_database(current_subject_id, volume_id, volume_name, volume_filepath)
```

just style, to communicate intent more clearly
assigning to `_` is a convention in python to indicate we are deliberately discarding a value